src/main/java/com/unzer/payment/Customer.java

```diff
@@ -48,7 +48,7 @@ public enum Salutation {
 	private String phone;
 	private String mobile;
 	private Address billingAddress;
-	private Address shippingAddress;
+	private ShippingAddress shippingAddress;
```

Comment:
This looks like a breaking change to me. That would not fit the version. Would it be possible to also allow Address object here and provide a setter for both address types?